packages/client/src/features/accessibility/keyboard-pointer/keyboard-pointer.ts

```diff
@@ -0,0 +1,112 @@
+/********************************************************************************
+ * Copyright (c) 2023 Business Informatics Group (TU Wien) and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+import { inject, injectable } from 'inversify';
+import {
+    AbstractUIExtension,
+    IActionDispatcher,
+    IActionHandler,
+    SModelRoot,
+    TYPES,
+    Action,
+    TriggerNodeCreationAction
+} from '~glsp-sprotty';
+import { SetKeyboardPointerRenderPositionAction } from './actions';
+import { KeyboardPointerMetadata } from './constants';
+import { KeyboardPointerKeyboardListener } from './keyboard-pointer.listener';
+import { KeyboardPointerPosition } from './keyboard-pointer.position';
+import { KeyboardGridCellSelectedAction } from '../keyboard-grid/action';
+import { EditorContextService } from '../../../base/editor-context-service';
+
+@injectable()
+export class KeyboardPointer extends AbstractUIExtension implements IActionHandler {
+    protected _triggerAction: TriggerNodeCreationAction = {
+        elementTypeId: 'task:automated',
+        kind: 'triggerNodeCreation'
+    };
+
+    position = new KeyboardPointerPosition(this);
+    keyListener;
```

Comment:
They should be protected I guess?
```suggestion
    protected position: KeyboardPointerPosition = new KeyboardPointerPosition(this);
    protected keyListener: KeyboardPointerKeyboardListener;
```